build-gluster-org/scripts/debian-ubuntu-package.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+#for both the debian and ubuntu packaging of various versions we have to
+#ssh to a particular ubuntu machine and package it using pbuilder.
+
+set -xe
+
+if [ $# -eq 5 ]; then
+        echo "building everything"
+        echo "packing debian distribution"
+        ../scripts/generic_package.sh debian stretch $SERIES $VERSION $RELEASE $LATEST_SERIES $LATEST_VERSION
+        ../scripts/generic_package.sh debian buster $SERIES $VERSION $RELEASE
+        ../scripts/generic_package.sh debian bullseye $SERIES $VERSION $RELEASE
+        echo "packing ubuntu distribution"
+        ../scripts/generic_package.sh ubuntu xenial $SERIES $VERSION $RELEASE
+        ../scripts/generic_package.sh ubuntu bionic $SERIES $VERSION $RELEASE
+        ../scripts/generic_package.sh ubuntu disco $SERIES $VERSION $RELEASE
+        ../scripts/generic_package.sh ubuntu eoan $SERIES $VERSION $RELEASE
+        ../scripts/generic_package.sh ubuntu focal $SERIES $VERSION $RELEASE
+fi
+if [ $# -gt 5 ]; then
+    if [ "$OS" == "all" ]; then
+        echo "packing all distribution"
+        echo "packing debian distribution"
+        ../scripts/generic_package.sh debian stretch $SERIES $VERSION $RELEASE $LATEST_SERIES $LATEST_VERSION
+        ../scripts/generic_package.sh debian buster $SERIES $VERSION $RELEASE
+        ../scripts/generic_package.sh debian bullseye $SERIES $VERSION $RELEASE
+        echo "packing ubuntu distribution"
+        ../scripts/generic_package.sh ubuntu xenial $SERIES $VERSION $RELEASE
+        ../scripts/generic_package.sh ubuntu bionic $SERIES $VERSION $RELEASE
+        ../scripts/generic_package.sh ubuntu disco $SERIES $VERSION $RELEASE
+        ../scripts/generic_package.sh ubuntu eoan $SERIES $VERSION $RELEASE
+        ../scripts/generic_package.sh ubuntu focal $SERIES $VERSION $RELEASE
+    elif [ "$OS" == "debian" ]; then
+        echo "packing debian alone"
+        if [ "$FLAVOR" == "stretch" ] || [ "$FLAVOR" == "9" ]; then
```

Comment:
I think it would be cleaner to have 1 single if with the 6 possibles values than 3 separate ones.

Reply:
can you please elaborate this comment

Reply:
Well, here we have 3 lines with if, they all run the same script with the same argument, the only difference is the echo. This seems harder to read while :

case $FLAVOR in
  "stretch" | "9" | "buster" | "10" | "bullseye" | "11") 
      ../scripts/generic_package.sh $OS $FLAVOR $SERIES $VERSION $RELEASE $LATEST_SERIES $LATEST_VERSION 

is shorter